met-web/src/components/userManagement/listing/AddUserModal.tsx

```diff
@@ -0,0 +1,177 @@
+                                justifyContent="flex-start"
+                                rowSpacing={4}
+                            >
+                                <Grid item xs={12}>
+                                    <MetLabel sx={{ marginBottom: '2px', display: 'flex' }}>
+                                        Select the user you want to add
+                                    </MetLabel>
+
+                                    <Controller
+                                        control={control}
+                                        name="user"
+                                        render={({ field: { ref, onChange, ...field } }) => (
+                                            <Autocomplete
+                                                options={users || []}
+                                                onChange={(_, data) => {
+                                                    onChange(data);
+                                                }}
+                                                renderInput={(params) => (
+                                                    <TextField
+                                                        {...params}
+                                                        {...field}
+                                                        inputRef={ref}
+                                                        fullWidth
+                                                        placeholder="(Select one)"
+                                                        error={Boolean(userErrors)}
+                                                        helperText={String(userErrors?.message || '')}
+                                                    />
+                                                )}
+                                                getOptionLabel={(user: User) => `${user.first_name} ${user.last_name}`}
+                                            />
+                                        )}
+                                    />
+                                </Grid>
+                                <Grid item xs={12}>
+                                    <FormControl error={Boolean(errors['group'])}>
+                                        <FormLabel
+                                            id="controlled-radio-buttons-group"
+                                            sx={{ fontWeight: 'bold', color: Palette.text.primary }}
+                                        >
+                                            What role would you like to assign to this user?
+                                        </FormLabel>
+                                        <ControlledRadiGroup name="group">
+                                            <FormControlLabel
+                                                value={USER_GROUP.ADMIN}
+                                                control={<Radio />}
+                                                label="Adminstrator"
```

Comment:
since you have a constant USER_GROUP.ADMIN , may be label is also something which we can put in constant?

Reply:
Good suggestion, done. And I'm hoping we would be able to pull these directly from KC, like the name and the title of each group at some point